pkg/install/templates/memcache-account.yaml.tmpl

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Why do we need a specific service account for memcached? Why not use the flux service account?

Reply:
We probably don't. I will remove